implementations/rust/ockam/ockam_command/src/node/start.rs

```diff
@@ -45,12 +43,15 @@ async fn run_impl(
     let node_name = get_node_name(&opts.state, &cmd.node_name);
 
     let node_state = opts.state.nodes.get(&node_name)?;
-    // Check if node is already running
-    if node_state.is_running() && !cmd.force {
-        println!(
-            "Restart aborted, node: {} already running",
-            node_state.name()
-        );
+    // Abort if node is already running
+    if node_state.is_running() {
+        opts.terminal
+            .stdout()
+            .plain(fmt_err!(
+                "Restart aborted, node '{}' already running",
```

Comment:
Maybe we could say that the node is already running and that the user should call `ockam node stop` then `ockam node start` if they want to restart the node?

Reply:
yes